purchase_security/models/purchase.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Tecnativa - Stefan Ungureanu
```

Comment:
Only one model per file, unless it's a "children" model, so this file should be called `purchase_team.py`, and only contains the first class. The other should be put in a file `purchase_order.py`. If you also inherit `purchase.order.line`, then that one can fit inside `purchase_order.py`.